lib/NodeManager.php

```diff
@@ -57,6 +65,10 @@ public function find($identifier)
             throw new DocumentNotFoundException(sprintf(
                 'Could not find document with ID or path "%s"', $identifier
             ), null, $e);
+        } catch (ItemNotFoundException $e) {
+            throw new DocumentNotFoundException(sprintf(
+                'Could not find document with ID or path "%s"', $identifier
```

Comment:
And we are already duplicating the exception message, because it's set in the wrong place 😜